features/backoffice_site_address.feature

```diff
@@ -0,0 +1,22 @@
+@backoffice
+Feature: Back office user registers a customers new waste exemption activity
```

Comment:
Could we make the feature description more relevant to the test i.e. try and put something in here as to why we need to capture the EA admin area